onnx-tool/src/pth_onnx_export.py

```diff
@@ -0,0 +1,83 @@
+import torch
+from torch import Tensor, nn
+import onnx
+import onnxruntime as rt
+import numpy as np
+
+
+class NeuralNetwork(nn.Module):
+    def __init__(self):
+        super(NeuralNetwork, self).__init__()
+        self.flatten = nn.Flatten()
+        self.linear = nn.Linear(64*3, 64*128 + 128)
```

Comment:
the linear layer requires quite a lot of state to store all its weights (it's still significantly less than the original models though), maybe we can even reduce this to no state at all. candidates for this might be `gather()` or `scatter()` or `tile()`. ideally we could also avoid any computation by just broadcasting, a candidate would be `expand()`. for all those options we would need to check if they can be consumed by tract. for example in our current bert models the (onnx) gather operation is used, but the onnx gather seems to be somewhat different from the torch gather (in terms of output dimensions), so again we would need to test if this is feasible.